private/versions.bzl

```diff
@@ -1,11 +1,11 @@
-_COURSIER_CLI_VERSION = "v2.0.0-RC5-3"
+_COURSIER_CLI_VERSION = "v2.0.6"
 
 COURSIER_CLI_HTTP_FILE_NAME = ("coursier_cli_" + _COURSIER_CLI_VERSION).replace(".", "_").replace("-", "_")
 COURSIER_CLI_GITHUB_ASSET_URL = "https://github.com/coursier/coursier/releases/download/{COURSIER_CLI_VERSION}/coursier.jar".format(COURSIER_CLI_VERSION = _COURSIER_CLI_VERSION)
 
 # Run 'bazel run //:mirror_coursier' to upload a copy of the jar to the Bazel mirror.
```

Comment:
@jin I don't believe I have access to upload to `mirror.bazel.build` can you upload this version?

Reply:
Yes, let me patch and apply

Reply:
@jin any ideas what's going wrong with Windows?  Unfortunately I don't have a Windows machine I can test on.

Reply:
Unfortunately I still haven't been able to get a Windows machine right now either :(

Reply:
Looks like calling `coursier` 2.0.6 without arguments on Windows was hanging on CI because it was awaiting input for
```
Checking if ~\AppData\Local\Coursier\data\bin is in PATH
  Should we add ~\AppData\Local\Coursier\data\bin to your PATH? [Y/n]
```
by adding `--help` to the `coursier` command it no longer asks to update the PATH and runs successfully.

Reply:
Nice catch!! Thank you. Do you mind adding a comment on the `--help` append to mention this issue?

Reply:
Done.  And updated to 2.0.7 since it was released since this PR was opened.  Can you update 2.0.7 to the google mirror?

Reply:
Done.

```
$ bazel run //:mirror_coursier
DEBUG: /usr/local/google/home/jingwen/.cache/bazel/_bazel_jingwen/8484bc4fff18ee4a905b69a9ddb0e143/external/bazel_toolchains/rules/rbe_repo/version_check.bzl:68:14:
Current running Bazel is ahead of bazel-toolchains repo. Please update your pin to bazel-toolchains repo in your WORKSPACE file.
DEBUG: /usr/local/google/home/jingwen/.cache/bazel/_bazel_jingwen/8484bc4fff18ee4a905b69a9ddb0e143/external/bazel_toolchains/rules/rbe_repo/checked_in.bzl:125:14: buildkite_config not using checked in configs; Bazel version 3.7.1 was picked/selected but no checked in config was found in map {"0.20.0": ["8.0.0"], "0.21.0": ["8.0.0"], "0.22.0": ["8.0.0", "9.0.0"], "0.23.0": ["8.0.0", "9.0.0"], "0.23.1": ["8.0.0", "9.0.0"], "0.23.2": ["9.0.0"], "0.24.0": ["9.0.0"], "0.24.1": ["9.0.0"], "0.25.0": ["9.0.0"], "0.25.1": ["9.0.0"], "0.25.2": ["9.0.0"], "0.26.0": ["9.0.0"], "0.26.1": ["9.0.0"], "0.27.0": ["9.0.0"], "0.27.1": ["9.0.0"], "0.28.0": ["9.0.0"], "0.28.1": ["9.0.0"], "0.29.0": ["9.0.0"], "0.29.1": ["9.0.0", "10.0.0"], "1.0.0": ["9.0.0", "10.0.0"], "1.0.1": ["10.0.0"], "1.1.0": ["10.0.0"], "1.2.0": ["10.0.0"], "1.2.1": ["10.0.0"], "2.0.0": ["10.0.0"], "2.1.0": ["10.0.0"], "2.1.1": ["10.0.0", "11.0.0"], "2.2.0": ["11.0.0"], "3.0.0": ["11.0.0"], "3.1.0": ["11.0.0"], "3.2.0": ["11.0.0"], "3.3.0": ["11.0.0"], "3.3.1": ["11.0.0"], "3.4.1": ["11.0.0"], "3.5.0": ["11.0.0"]}
INFO: Analyzed target //:mirror_coursier (13 packages loaded, 49 targets configured).
INFO: Found 1 target...
Target //:mirror_coursier up-to-date:
  bazel-bin/mirror_coursier
INFO: Elapsed time: 0.899s, Critical Path: 0.03s
INFO: 3 processes: 3 internal.
INFO: Build completed successfully, 3 total actions
INFO: Build completed successfully, 3 total actions

Copying file://external/coursier_cli_v2_0_7/file/downloaded [Content-Type=application/octet-stream]...
- [1 files][ 34.0 MiB/ 34.0 MiB]
Operation completed over 1 objects/34.0 MiB.
```